transforms/universal/fdedup_multi_step/kfp_ray/Makefile_ori

```diff
@@ -0,0 +1,51 @@
+REPOROOT=${CURDIR}/../../../../
```

Comment:
I assume this file is `Makefile original`. Therefore we need to rename it back or remove it and create a new Makefile.